research/metrics/index.md

```diff
@@ -18,16 +18,103 @@ The PCMDI Metrics Package ([PMP][pmp]) is a capability that is used to produce a
+
+<p align="right"><a href="#top">Back to List</a></p>
+
+---
+## <a name="updates"></a>Update History
+- [**v1.5.1**][v1.5.1] (2022-11-04): Mean climate interactive portrait plot updated and precipitation variability across timescale portrait plot newly added.
+- [**v1.5.0**][v1.5.0] (2020-10-08): Precipitation benchmarking newly added and Mean climate parallel coordinate and portrait plots updated.
+- [**v1.4.1**][v1.4.1] (2020-07-20): MJO metrics with recent CMIP6 results
+- [**v1.4.0**][v1.4.0] (2020-07-10): ENSO Metrics updated with Interactive Portrait Plot with recent CMIP6 results
+- [**v1.3.2**][v1.3.2] (2020-06-19): Mean climate summaries updated with recent CMIP6 results with OBS info updated using PCMDIobs2
+- [**v1.3.1**][v1.3.1] (2019-10-07): Mean climate summaries updated with recent CMIP6 results
+- [**v1.3.0**][v1.3.0] (2019-09-06): ENSO metrics added
+- [**v1.2.0**][v1.2.0] (2019-08-29): Mean climate metrics added
+- [**v1.1.0**][v1.1.0] (2019-07-18): MJO metrics added
+- [**v1.0.0**][v1.0.0] (2019-06-20): Initial public release
+- [**v1.0.0-beta**][v1.0.0-beta] (2019-06-18): Monsoon precipitation onset, decay, and duration (CMIP5) added
+- [**v1.0.0-alpha**][v1.0.0-alpha] (2019-05-31): Test release: Extratropical Modes of Variability (CMIP5 and CMIP6)
+
+<p align="right"><a href="#top">Back to List</a></p>
+
+---
+
+
 
 Questions or comments about the PCMDI Simulation Summaries should be sent to the [PMP team](mailto:pcmdi-metrics@llnl.gov).
 
 The efforts of the authors are supported by the [Regional and Global Climate Modeling Program][RGMA] of the [United States Department of Energy's Office of Science][DOEOS]. This work is funded by the Climate and Environmental Sciences Division of the DOE Office of Science and is performed under the auspices of the U.S. Department of Energy by [Lawrence Livermore National Laboratory][LLNL] under contract DE-AC52-07NA27344. LLNL-WEB-812310
 
+[latest]: {{site.baseurl}}/research/metrics/
+
+[v1.5.1]: {{site.baseurl}}/research/metrics/v1.5.0
+[v1.5.0]: {{site.baseurl}}/research/metrics/v1.5.0
+[v1.4.1]: {{site.baseurl}}/research/metrics/v1.4.1
+[v1.4.0]: {{site.baseurl}}/research/metrics/v1.4.0
+[v1.3.2]: {{site.baseurl}}/research/metrics/v1.3.2
+[v1.3.1]: {{site.baseurl}}/research/metrics/v1.3.1
+[v1.3.0]: {{site.baseurl}}/research/metrics/v1.3.0
+[v1.2.0]: {{site.baseurl}}/research/metrics/v1.2.0
+[v1.1.0]: {{site.baseurl}}/research/metrics/v1.1.0
+[v1.0.0]: {{site.baseurl}}/research/metrics/v1.0.0
+[v1.0.0-beta]: {{site.baseurl}}/research/metrics/v1.0.0-beta
+[v1.0.0-alpha]: {{site.baseurl}}/research/metrics/v1.0.0-alpha
+
+[description_mean_clim]: mean_clim
+[description_variability]: variability_modes
+[description_monsoon]: monsoon
```

Comment:
Needs a link pointing to latest/monsoon.

Reply:
@mauzey1 Thank you for pointing out. I just added it.